reana_commons/config.py

```diff
@@ -224,6 +225,35 @@
 )
 """Name of the shared CEPHFS PVC which will be used by all REANA jobs."""
 
+REANA_JOB_HOSTPATH_MOUNTS = json.loads(os.getenv("REANA_JOB_HOSTPATH_MOUNTS", "[]"))
+"""List of dictionaries composed of name, hostPath and mountPath.
+
+- ``name``: name of the mount.
+- ``hotsPath``: path to the directory in the Kubernetes nodes to be mounted in the jobs.
```

Comment:
- `hostPath` typo

I would shorten the explanation and use "pods" to make it clearer, for example like:

- hostPath: path in the Kubernetes cluster host nodes that will be mounted into job pods
- mountPath: path inside job pods where hostPath will get mounted

Reply:
Thanks, much clearer this way :)